src/index.js

```diff
@@ -1,4 +1,61 @@
-import hello from './modules/hello.js';
 import './style.css';
+import registerNewGame from './modules/game.js';
 
-hello();
\ No newline at end of file
+registerNewGame('monicahs');
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/c0Kl9crOIDTmH3nrtoHL/scores/';
+
+const form = document.querySelector('.data-form');
+const refreshBtn = document.querySelector('.btn-refresh');
+const postData = (name, scores) => {
+  fetch(url, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      user: name,
+      score: scores,
+    }),
+  })
+    .then(() => {
+      refreshBtn.click();
+    });
+};
```

Comment:
- Please use `async await` with `fetch` when dealing (both send and receive) with the API as project [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md#:~:text=Use%20async%20and%20await%20JavaScript%20features%20to%20consume%20the%20API.) dictates. 😄